src/core/distributions/phylogenetics/tree/birthdeath/BirthDeathSamplingTreatmentProcess.cpp

```diff
@@ -604,7 +612,7 @@ bool BirthDeathSamplingTreatmentProcess::countAllNodes(void) const
               int at_event = whichIntervalTime(t);
 
               // If this bifurcation is not at an event time (and specifically at an event time with Lambda[i] > 0), it's a serial bifurcation
-              if ( at_event == -1 )
+              if ( at_event == -1 || lambda_event[at_event] < DBL_EPSILON)
```

Comment:
why was this removed? I had some tests where the burst events were not seen as burst events because the times didn't exactly match due to rounding. Is this now covered better in at_event?

Reply:
I didn't realize this was an issue, sure, let's keep it

Reply:
The patch is adding something... I guess @hoehna is asking why it was original removed?